### ISSUE TITLE
Please Add Support for Muilding Multi-Arch Images

### ISSUE THREAD

**Issue by alexgaganashvili** [concourse/docker-image-resource#356]
### Is your feature request related to a problem? Please describe.

.

### Describe the solution you'd like

I'd like the resource to support building multi-arch images.

**Comment by xtremerui** [comment]
The feature is available on https://github.com/concourse/registry-image-resource. Check `platform` in its source configuration

**Comment by alexgaganashvili** [comment]
@xtremerui, yes, I'm aware of that feature being available in registry-image-resource. However, it does not support multiple architectures.
